python/cloudtik/runtime/spark/scripts/install.sh

```diff
@@ -156,8 +156,13 @@ function install_spark_with_cloud_jars() {
 }
 
 function install_hdfs_fuse() {
-    # install fuse_dfs
-    :
+    if [ -z "${HADOOP_VERSION}" ]; then
+        HADOOP_VERSION=3.3.1
+    fi
+    sudo wget  -q --show-progress https://d30257nes7d4fq.cloudfront.net/downloads/hadoop/fuse_dfs-${HADOOP_VERSION} -O /usr/bin/fuse_dfs
+    sudo wget  -q --show-progress https://d30257nes7d4fq.cloudfront.net/downloads/hadoop/fuse_dfs_wrapper-${HADOOP_VERSION}.sh -O /usr/bin/fuse_dfs_wrapper.sh
```

Comment:
fuse_dfs-${HADOOP_VERSION} is an executable binary. Let's consider the arch for any architecture dependent binaries like what we do for ganglia modpython.so. (No need for .sh). 

        arch=$(uname -m)
        wget -q https://d30257nes7d4fq.cloudfront.net/downloads/ganglia/modpython-${arch}.so -O /tmp/modpython.so

upload the file in the format of fuse_dfs-${HADOOP_VERSION}-$(arch)

Reply:
Have done. Please help to merge.